example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -509,7 +515,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				INFOPLIST_FILE = Runner/Info.plist;
+				INFOPLIST_FILE = "Runner/Info-Debug.plist";
```

Comment:
I'm assuming the `-Debug.plist` was added so we can debug locally?

(it's one of those things we should maybe have in a separate PR).